src/material.cc

```diff
@@ -53,6 +53,34 @@ void Material::Record(Context* ctx) const {
   comp_->Record(ctx);
 }
 
+std::string Material::GetNormalizedCompAtom() const {
```

Comment:
The bulk of this implementation should probably for a `Composition` that gets called from this `Material` method.

A method like this is commonly named `to_string()`, but maybe in this case `to_atom_string()` or `to_string(atom|mass)`?